src/Build/Evaluation/LazyItemEvaluator.cs

```diff
@@ -151,15 +151,15 @@ public ItemData Clone(IItemFactory<I, I> itemFactory, ProjectItemElement initial
 
         private class MemoizedOperation : IItemOperation
```

Comment:
I'd add a summary here.

Reply:
That's out of scope in my opinion.

Reply:
If you gained enough context on this class I'd still suggest writing a summary, but it is out of scope. I don't feel too strongly about it.